package.json

```diff
@@ -67,16 +67,16 @@
 		"eslint-plugin-chai-expect": "^2.2.0",
 		"husky": "^4.3.8",
 		"lint-staged": "^11.0.0",
-		"markdownlint-cli2": "^0.3.0",
-		"mocha": "^8.2.1",
+		"markdownlint-cli2": "^0.4.0",
+		"mocha": "^9.2.0",
```

Comment:
Test failures seem related to this version upgrade – can you undo this?

Reply:
done.